containers/Ideas/Idea.js

```diff
@@ -124,6 +128,43 @@ export const Idea = (props) => {
     return await fetchIdea();
   };
 
+  const handleCommentSubmit = (values) => {
+    AsyncStorage.getItem('authToken')
+      .then((token) => {
+        return API.addComment(objectToComment.contentType, objectToComment.pk, values, token);
+      })
+      .then((response) => {
+        const {statusCode, data} = response;
+        if (statusCode == 201) {
+          if (data.content_type == data.comment_content_type) {
+            setCommentLastCommented(data.object_pk);
+          }
+          else {
+            setCommentLastCommented(-1);
+          }
+          fetchComments(idea.content_type, idea.pk);
+          setObjectToComment({'contentType': idea.content_type, 'pk': idea.pk});
+        }
+        else {
+          const errorMessage = 'That did not work.';
+          let errorDetail;
+          if (statusCode==403) {
+            errorDetail = data.detail;
+          }
+          else if (statusCode == 400) {
+            errorDetail = ('comment' in data ? ('Comment: ' + data['comment']) : 'Bad request');
+          }
+          Alert.alert(errorMessage, errorDetail, [{ text: 'Ok' }]);
+        }
+      });
+
+  };
+
+  const handleCommentReply = (commentContentType, commentObjectPk) => {
+    setObjectToComment({'contentType': commentContentType, 'pk': commentObjectPk});
+    setFocusCommentForm(true);
```

Comment:
@khamui funny thing is that when you intialize FocusCommentForm with true and then set to false here, it looses focus when clicking on reply, so the other way around it works :shrug:

Reply:
I didn't manage to make the reply request work. it throws an error.
Somehow the contenttype is undefined:
<img width="966" alt="Bildschirmfoto 2022-01-25 um 16 13 17" src="https://user-images.githubusercontent.com/52801593/151014990-fb4ee13f-bf4a-475b-b3b8-b47de6305f32.png">